digitalpy/core/persistence/impl/null_mapper.py

```diff
@@ -1,11 +1,11 @@
 from typing import Any, List
-from digitalpy.model.persistence_mapper import PersistenceMapper
-from digitalpy.model.build_depth import BuildDepth
-from digitalpy.model.persistent_object import PersistentObject
-from digitalpy.translation.message import Message
-from digitalpy.model.persistence_operation import PersistenceOperation
-from digitalpy.model.paging_info import PagingInfo
-from digitalpy.model.object_id import ObjectId
+from digitalpy.core.persistence.persistence_mapper import PersistenceMapper
```

Comment:
**[pep8]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>I003 isort expected 1 blank line in imports, found 0